docs/content.zh/docs/deployment/resource-providers/native_kubernetes.md

```diff
@@ -97,14 +97,34 @@ COPY /path/of/my-flink-job.jar $FLINK_HOME/usrlib/my-flink-job.jar
 After creating and publishing the Docker image under `custom-image-name`, you can start an Application cluster with the following command:
 
 ```bash
+# Local Schema
 $ ./bin/flink run-application \
     --target kubernetes-application \
     -Dkubernetes.cluster-id=my-first-application-cluster \
     -Dkubernetes.container.image=custom-image-name \
     local:///opt/flink/usrlib/my-flink-job.jar
+
+# FileSystem
+$ ./bin/flink run-application \
+    --target kubernetes-application \
+    -Dkubernetes.cluster-id=my-first-application-cluster \
+    -Dkubernetes.container.image=custom-image-name \
+    s3://my-bucket/my-flink-job.jar
+
+# Http/Https Schema
+$ ./bin/flink run-application \
+    --target kubernetes-application \
+    -Dkubernetes.cluster-id=my-first-application-cluster \
+    -Dkubernetes.container.image=custom-image-name \
+    http://ip:port/my-flink-job.jar
 ```
+{{< hint info >}}
+Now, The jar package supports reading from the [flink filesystem]({{< ref "docs/deployment/filesystems/overview" >}}#docker-hub-flink-images) or Http/Https in Application Mode.  
+The jar package will be downloaded from filesystem to
+[kubernetes.user.artifacts.base.dir]({{< ref "docs/deployment/config" >}}#kubernetes-user-artifacts-base-dir)/[kubernetes.namespace]({{< ref "docs/deployment/config" >}}#kubernetes-namespace)/[kubernetes.cluster-id]({{< ref "docs/deployment/config" >}}#kubernetes-cluster-id) path in image.
+{{< /hint >}}
+<span class="label label-info">Note</span> `local` schema is also supported .
```

Comment:
if we use the non-local URI, we do not have to set the init container in pod template (as shown in [link](https://github.com/apache/flink/blob/dca656b922b6c6d350301d8b8f3e60f87b03c381/docs/content.zh/docs/deployment/resource-providers/native_kubernetes.md#example-of-pod-template)) Or we can add comments here to clarify that: if a local URI provided, the jar must be provided in the image or download by a init container.

Reply:
Besides, I think the `file` schema is not supported (because it's in the client side)

Reply:
According to the flink document [https://nightlies.apache.org/flink/flink-docs-release-1.15/zh/docs/deployment/filesystems/overview/](url). `file` schema refers to a file in the local file system.I think `flle` schema  should be supported.

Reply:
But, I think the `local` points to the file in the container, the `file` points to the jar in the client side. If it's in the client side, we can not get it from the jobmanager container, Right ?

Reply:
`file` schema is pointing to the jar in the container. In this case , `local` schema is equal to `file` schema. The difference between `file` and `local` is that `local` will return the jar path directly. We will fetch the jar  from  the path `file`  points to 
to the path that  `kubernetes-user-artifacts-base-dir`/`kubernetes-namespace`/`kubernetes-cluster-id`.
For this ,I have tested.
![fd74ef22a76b5f894b2072484ad9eff](https://user-images.githubusercontent.com/21351694/186449105-61319671-2d6c-478f-a49d-7b68e952bf21.png)
![00e6f8cb62992bf0bbf413feece51de](https://user-images.githubusercontent.com/21351694/186449122-f7a9dddf-d1de-4f49-ae30-1720e174b027.png)
![25939ce0cc994d156d93953e34e7b4c](https://user-images.githubusercontent.com/21351694/186449136-42c88a23-a7e8-4875-b179-963b65790453.png)

Reply:
From the user side, I think the `local` is different from `file`. My point is that, although we support the filesystem URL, but we still not support user specify a jar on the client side. 

For example, If I want to submit `job.jar` which is at client machine's `/tmp/` dir, It's not supported. I guess this is why we have `local` and `file` two different scheme here. Also cc @wangyang0918 

```
./bin/flink run-application \
    --target kubernetes-application \
    -Dkubernetes.cluster-id=my-first-application-cluster \
    -Dkubernetes.container.image=custom-image-name \
    /tmp/job.jar  
```

Reply:
Exactly as you said ,`/opt/job.jar` is equal `file:///opt/job.jar`.This makes it difficult for users to distinguish whether the Jar package is on the client machine or the container.
BTW, are there plans to support submitting user jars from the client? If `kubernetes.rest-service.exposed.type` is set to ClusterIP, it is difficult client connect the jobmanager deployed in k8s.